pkg/services/sqlstore/user_auth_test.go

```diff
@@ -16,7 +16,7 @@ func TestUserAuth(t *testing.T) {
 	Convey("Given 5 users", t, func() {
 		var err error
 		var cmd *m.CreateUserCommand
-		users := []m.User{}
+		// users := []m.User{}
```

Comment:
could you describe the fix here, unused variable?

Reply:
Yes, should I remove it?

Reply:
yes